attribute.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+# Represents a classroom.
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    students << student
+    student.classroom = self
+  end
+end
+
+# Represents a Student.
+class Student
+  attr_accessor :name, :classroom
+
+  def initialize(name)
+    @name = name
+    @classroom = nil
+  end
+end
+
+# Represents a Book.
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+end
```

Comment:
[OPTIONAL] - According to the [project requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_associations.md):

Kindly create the `has-many` side (a book has many rentals) 

A method for adding rentals should be present, the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 👀

Link to the [example in a previous lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)